src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/ProjectConfigurationStateSynchronizerTest.cs

```diff
@@ -19,7 +18,6 @@
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer
 {
-    [SuppressMessage("Usage", "VSTHRD002:Avoid problematic synchronous waits", Justification = "Requires execution on the foreground thread.")]
```

Comment:
Interesting that this is fixed by this change?

Reply:
I don't think it is, but I randomly removed it and it didn't seem to produce an error anymore